### PR TITLE
M sc1 59 19

### PR DESCRIPTION
**Pull request by mrdemqn**

### CODE REVIEW

File: dkakhanouski/01_HTML/Menu.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+}
+li a:hover {
+  color: grey;
+  background-color: white;
+  }
+@media all and (min-width: 900px) {
+  ul, li{
+    min-width: 150px;
+    width: auto;
+    display: inline-block;
```
Comment (pluhin) [inline dkakhanouski/01_HTML/Menu.html:46]: or display inline block, or float, but just one

File: dkakhanouski/01_HTML/Menu_Add.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+li ul li:nth-of-type(1) {
+  margin-left: 0px;
+}
+li ul li:nth-of-type(2) {
+  margin-left: 0px;
+}
+li ul li:nth-of-type(3) {
+  margin-left: 0px;
+}
+li:hover ul {
```
Comment (pluhin) [inline dkakhanouski/01_HTML/Menu_Add.html:26]: Is isn't good solution for smooth appearance of nested lists, you have hover zones under day of the week
nested list should be appear when we do hover just on the particular day

**Review by pluhin — CHANGES_REQUESTED** [review]

File: dkakhanouski/01_HTML/Menu_Add.html (hunk 2 above)
Comment (mrdemqn) [reply dkakhanouski/01_HTML/Menu_Add.html]: I have fixed it, thanks.

**Comment by mrdemqn** [comment]
I have fixed it, thanks.

**Review by pluhin — APPROVED** [review]

**Pull request merged**